examples/ovs-cni.yml

```diff
@@ -15,6 +15,8 @@ spec:
       labels:
         tier: node
         app: ovs-cni
+      annotations:
+        description: Ovs-cni pod configures Kubernetes networks on top of Open vSwitch bridges available on nodes
```

Comment:
OVS CNI configures

We usually write it as OVS CNI.

And there is no reason to say that it is a pod

Reply:
Done